src/sage/rings/function_field/drinfeld_modules/morphism.py

```diff
@@ -540,12 +540,12 @@ def __invert__(self):
             sage: K.<z> = Fq.extension(3)
             sage: coeffs = [z] + [K.random_element() for _ in range(10)]
             sage: phi = DrinfeldModule(A, coeffs)
-            sage: f = phi.hom(K.random_element())
-            sage: g = ~f
+            sage: f = phi.hom(K.random_element()) # known bug: macos (sporadic ValueError: the input does not define an isogeny)
+            sage: g = ~f # known bug: macos (sporadic ValueError: the input does not define an isogeny)eny
```

Comment:
```suggestion
            sage: f = phi.hom(K.random_element())  # known bug: macos (sporadic ValueError: the input does not define an isogeny)
            sage: g = ~f  # known bug: macos (sporadic ValueError: the input does not define an isogeny)
```

per codestyle, 2 spaces before the `#` (please fix throughout)